askama_derive/src/generator.rs

```diff
@@ -727,6 +727,42 @@ impl<'a> Generator<'a> {
                 args.len()
             )));
         }
+        let mut output_args = Vec::new();
+        let args = if args
+            .iter()
+            .any(|arg| matches!(arg, Expr::NamedArgument(_, _)))
+        {
+            // First we check that all named arguments actually exist in the called item.
+            for arg in args.iter() {
+                if let Expr::NamedArgument(arg_name, _) = arg {
```

Comment:
Let's do `let .. else` for the outer branch here?

Reply:
I don't really see how it would improve the code since there is no `else` clause here?

Reply:
It reduces rightward drift for the further check, so I would still like to do this.

Reply:
Now there is a good reason for it (the `break` which was added only recently) so adding it!